src/main.c

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
.cpp would not compile right now with Zephyr 1.6 unless we change some of the Zephyr files